util/imageutil/buildinfo.go

```diff
@@ -9,6 +9,8 @@ import (
 )
 
 // BuildInfo returns build info from image config.
+// Deprecated: Build information format has been deprecated and will be removed
+// in a future release. Use provenance attestations instead.
```

Comment:
I _think_ the correct format is to have 1 empty line before;

```suggestion
// BuildInfo returns build info from image config.
//
// Deprecated: Build information format is deprecated and will be removed
// in a future release. Use provenance attestations instead.
```

You can also run `pkg.go.dev` locally with;

```bash
pkgsite -http :6060 .
```

Install it from https://pkg.go.dev/golang.org/x/pkgsite@v0.0.0-20230201160136-f93c7b3d884a/cmd/pkgsite